typescript/servicecatalog/portfolio-with-ec2-product/README.md

```diff
@@ -0,0 +1,58 @@
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Developer Preview](https://img.shields.io/badge/stability-Developer--Preview-important.svg?style=for-the-badge)
+
+> **This is an experimental example. It may not build out of the box**
+>
+> This example is built on Construct Libraries marked "Developer Preview" and may not be updated for latest breaking changes.
+>
+> It may additionally requires infrastructure prerequisites that must be created before successful build.
+>
+> Non-core construct in preview used in this example: `@aws-cdk/aws-servicecatalog-alpha` 
+>
+> If build is unsuccessful, please create an [issue](https://github.com/aws-samples/aws-cdk-examples/issues/new) so that we may debug the problem 
+---
+<!--END STABILITY BANNER-->
+
+# Using Product Stacks with AWS Service Catalog (VPC with EC2 Instance)
+
+This is a CDK example of a [Product Stack](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-servicecatalog.ProductStack.html).
+
+It creates a Service Catalog Portfolio and associated Product containing template for the following resources:
+* A VPC
+* A public subnet
+* A security group
+* An EC2 instance in the subnet
+
+Additionally the Portfolio contains:
+* A [Launch Constraint](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints-launch.html) on the EC2 instance size
+* A [Notification Constraint](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints-notification.html) on stack events
```

Comment:
Like the Python README let's include a note about how this project uses an experimental API: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_servicecatalog-readme.html